### PR TITLE
fix: use `--variant` CLI option in sidebar install instructions

### PR DESCRIPTION
**Pull request by aaronsteers**
Sample rendering:

- [default](https://deploy-preview-774--meltano-hub-gridsome.netlify.app/extractors/tap-postgres--transferwise/)
- [non-default](https://deploy-preview-774--meltano-hub-gridsome.netlify.app/extractors/tap-postgres--singer-io/)

### CODE REVIEW

**Comment by aaronsteers** [comment]
`is_default` is not yet passed to the sidebar component. After that is in place, this can exist draft status.

File: src/components/PluginSidebar.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="single-plugin-aside">
     <h4>Install</h4>
-    <pre><code>meltano add {{plugin_type}} {{ name }}</code></pre>
+    <pre><code>meltano add {{plugin_type}} {{ name }} <span v-if="!is_default">--variant {{ variant }}</span></code></pre>
```
Comment (aaronsteers) [inline src/components/PluginSidebar.vue:4]:
```suggestion
    <pre><code>meltano add {{plugin_type}} {{ name }}<span v-if="!is_default"> --variant {{ variant }}</span></code></pre>
```

**Comment by aaronsteers** [comment]
@rwfeather - Ready for review.

**Review by rwfeather — APPROVED** [review]
lgtm

**Pull request merged**